auslib/blobs/base.py

```diff
@@ -56,6 +56,7 @@ def createBlob(data):
 
 class Blob(dict):
     jsonschema = None
+    hashLen = {'sha256': 64, 'sha384': 96, 'sha512': 128}
```

Comment:
This probably doesn't belong on the Blob class. Probably best to put it somewhere in auslib/util/.